task/options/options.go

```diff
@@ -45,7 +46,18 @@ type Duration struct {
 }
 
 func (a Duration) String() string {
-	return ast.Format(&a.Node)
+	// NOTE: This is a copy of `formatDurationLiteral` from the flux codebase.
```

Comment:
Hi Daniel, 
I am adding an exportable method that can be used for formatting Duration. Changes are in the review atm. https://github.com/influxdata/flux/pull/4009